bucket/tortoisehg.json

```diff
@@ -0,0 +1,74 @@
+{
+    "version": "5.9.3",
+    "description": "TortoiseHg. A set of graphical tools and a shell extension for the Mercurial distributed revision control system.",
+    "homepage": "https://tortoisehg.bitbucket.io/",
+    "license": {
+        "identifier": "GPL-2.0-only",
+        "url": "https://tortoisehg.readthedocs.io/en/latest/preface.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-5.9.3-x64.msi",
+            "hash": "e17c1f81590a25bfe02b2adf4733bf0b0d58b5556833236ec70632b2d6a7b633"
+        },
+        "32bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-5.9.3-x86.msi",
+            "hash": "1e254c2e2a176ff4aad2e88259d926ebc74117ba0ba1d7f61e89220a68b34ede"
+        }
+    },
+    "extract_dir": "PFiles\\TortoiseHg",
+    "bin": [
+        "thg.exe",
+        "thgw.exe",
+        "hg.exe",
+        "Pageant.exe",
+        "TortoiseHgOverlayServer.exe",
+        "lib\\kdiff3.exe",
+        "lib\\TortoisePlink.exe"
+    ],
+    "shortcuts": [
+        [
+            "thg.exe",
+            "Thg, TortoiseHg's GUI tools for Mercurial SCM (Hg)"
+        ],
+        [
+            "thgw.exe",
+            "TortoiseHg Workbench"
+        ],
+        [
+            "hg.exe",
+            "Mercurial Distributed SCM"
+        ],
+        [
+            "Pageant.exe",
+            "Pageant is a PuTTY authentication agent"
+        ],
+        [
+            "TortoiseHgOverlayServer.exe",
+            "Service which gives us icons on the version-controlled files or folders"
+        ],
+        [
+            "lib\\kdiff3.exe",
+            "Tool for Comparison and Merge of Files and Directories"
+        ],
+        [
+            "lib\\TortoisePlink.exe",
+            "Command-line connection utility (based on PuTTY Plink)"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/",
+        "regex": "tortoisehg-([\\d.]+)-x[468]{2}\\.msi"
+    },
+    "autoupdate": {
+    "note": "Thanks for using autoupdate, please test your updates!",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-$version-x64.msi"
+        },
+        "32bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-$version-x86.msi"
+        }
+    }
```

Comment:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-$version-x64.msi"
            },
            "32bit": {
                "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-$version-x86.msi"
            }
        }
```

indentation

Reply:
I don't see any benefit by adding the indentation.  The `architecture` tag is correctly indented.

Reply:
It is a nested object, it needs to be indented. 
![image](https://user-images.githubusercontent.com/46838874/145442756-fc91d775-76e8-44d5-95b3-f60b75da4400.png)

Reply:
I know that and I agree.  The point I'm trying to make is that I already see it indented.

![image](https://user-images.githubusercontent.com/731154/145605777-3dbf91fc-a243-492e-b60a-a2000b0ae52b.png)

Reply:
Do you not notice the second last and third last closing bracket? They are in the same column.

Reply:
@rashil2000 ah, sorry about did not notice :).  Thank you.